src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.CodeGeneration.targets

```diff
@@ -103,7 +103,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       UseServer="$(UseRazorBuildServer)"
       ForceServer="$(_RazorForceBuildServer)"
       PipeName="$(_RazorBuildServerPipeName)"
-      Version="$(RazorLangVersion)"
+      Version="$(ResolvedRazorLangVersion)"
```

Comment:
Do we super need this new variable? or just for consistency/debugging?

Reply:
Should our various project system implementations start consuming `ResolvedRazorLangVersion` now if it's available over `RazorLangVersion`?

Reply:
A bit of both. We already have `ResolvedRazorConfiguration`, `ResolvedRazorCompileToolset` so it made sense to follow the pattern. Plus it's easier to look in the build logs than scan for places the property was modified.

@NTaylorMullen where do we use `RazorLangVersion` in code today? I'd be happy to update this PR \ send a follow up to use the new property.

Reply:
If it's not required I'd really prefer to change this: ie don't add a new variable

Reply:
> A bit of both. We already have `ResolvedRazorConfiguration`, `ResolvedRazorCompileToolset` so it made sense to follow the pattern. Plus it's easier to look in the build logs than scan for places the property was modified.

This is concerning. All of our project systems (VSWindows, VSMac, VSCode) are definitely not using that 😨 . See below for a list of all the various impls. In VS Windows case the configuration bits are in a separate location:

- [RazorConfiguration.xaml](https://github.com/aspnet/AspNetCore-Tooling/blob/25fe68e2bc95d69154be35e96219cc978241f146/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Rules/RazorConfiguration.xaml)
- [RazorConfiguration.cs](https://github.com/aspnet/AspNetCore-Tooling/blob/25fe68e2bc95d69154be35e96219cc978241f146/src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorConfiguration.cs)


> @NTaylorMullen where do we use `RazorLangVersion` in code today? I'd be happy to update this PR \ send a follow up to use the new property.

- [RazorGeneral.xaml](https://github.com/aspnet/AspNetCore-Tooling/blob/25fe68e2bc95d69154be35e96219cc978241f146/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Rules/RazorGeneral.xaml#L20-L26)
- [RazorGeneral.cs](https://github.com/aspnet/AspNetCore-Tooling/blob/25fe68e2bc95d69154be35e96219cc978241f146/src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorGeneral.cs#L31)
- [DefaultRazorProjectHost.cs](https://github.com/aspnet/AspNetCore-Tooling/blob/master/src/Razor/src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/ProjectSystem/DefaultRazorProjectHost.cs#L108) (VS4Mac)
- [VSCode's LatestProjectConfigurationProvider](https://github.com/aspnet/Razor.VSCode/blob/c45160be9ee1e3176cb4c85c9612fcaa090b6b82/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/LatestProjectConfigurationProvider.cs#L133)

Reply:
> This is concerning

Filed an issue to track this: https://github.com/aspnet/AspNetCore/issues/8658

Out of curiosity, do we expect any of these tooling bits to work with an older version of the SDK? VS wouldn't, but I'm not super sure about VS4Mac or VSCode.

Reply:
Yes they should. In tooling we should be using latest and greatest if it's available but if not, fallback to older.

Reply:
Ok. That does limit what name we could use here. `ResolvedRazorLangVersion` isn't going to be available in older versions of the SDK. I'll use the existing name for now, we do have the ability to tweak in the future it the need arises.